tests/e2e/src/eth_getBalance.test.js

```diff
@@ -0,0 +1,165 @@
+import goto from "./goto";
+import testDataManage from "./create_test_data/createTestDataManage";
+
+let testDataInfo = null;
+const pageName = "eth_getBalance.html";
+describe("eth_getBalance", () => {
+  testDataInfo = testDataManage.readTestDataAsJson("testData_1.json");
```

Comment:
use `beforeAll` to allow skipping these preparation if we don't need these test cases.